composeApp/src/main/res/values-it/strings.xml

```diff
@@ -40,4 +41,13 @@
         <item quantity="other">%1$d fusi orari</item>
     </plurals>
     <string name="timezone_search_placeholder">Città, continente, id</string>
+    <string name="no_timezone">Nessun fuso orario</string>
+
+    <string name="create_viewing_time_just_started">Appena iniziato</string>
```

Comment:
Same naming issue with the keys. Also, should this be specific to movies? In languages with gendered nounes, movie and episodes might have opposing grammatical genders. It's just a coincidence they don't in italian.

E.g. for Spanish:
- Película -> feminine
- Episodio -> masculine